lib/streamlit/elements/file_uploader.py

```diff
@@ -8,53 +8,64 @@
 
 
 class FileUploaderMixin:
-    def file_uploader(dg, label, type=None, key=None, **kwargs):
+    def file_uploader(
+        dg, label, type=None, accept_multiple_files=False, key=None, **kwargs
+    ):
         """Display a file uploader widget.
-
         By default, uploaded files are limited to 200MB. You can configure
         this using the `server.maxUploadSize` config option.
-
         Parameters
         ----------
         label : str or None
             A short label explaining to the user what this file uploader is for.
         type : str or list of str or None
             Array of allowed extensions. ['png', 'jpg']
             By default, all extensions are allowed.
-        encoding : str or None
-            The encoding to use when opening textual files (i.e. non-binary).
-            For example: 'utf-8'. If set to 'auto', will try to guess the
-            encoding. If None, will assume the file is binary.
+        accept_multiple_files : bool
+                        Determines if multiple files can be uploaded at once.
+                        Default: False
         key : str
             An optional string to use as the unique key for the widget.
             If this is omitted, a key will be generated for the widget
             based on its content. Multiple widgets of the same type may
             not share the same key.
-
         Returns
         -------
-        BytesIO or StringIO or or list of BytesIO/StringIO or None
-            If no file has been uploaded, returns None. Otherwise, returns
-            the data for the uploaded file(s):
-            - If the file is in a well-known textual format (or if the encoding
-            parameter is set), the file data is a StringIO.
-            - Otherwise the file data is BytesIO.
-            - If multiple_files is True, a list of file data will be returned.
-
-            Note that BytesIO/StringIO are "file-like", which means you can
-            pass them anywhere where a file is expected!
-
+        None or UploadedFile or list of UploadedFile
+                        If no file has been uploaded, returns None. Otherwise, returns
```

Comment:
Spacing seems weird here. Intentional?

Reply:
It is not, I have it addressed in a docstring branch!